src/Microsoft.HttpRepl.Tests/OpenApi/EndpointMetadataReaderTests.cs

```diff
@@ -0,0 +1,495 @@
+using System.Collections.Generic;
```

Comment:
nit: copy the copyright header from other source files

Reply:
Added here:
https://github.com/aspnet/HttpRepl/commit/6f329922585d50ba40849b897f9da2bbf50fa4f7